tests/std/tests/P0980R1_constexpr_strings/test.cpp

```diff
@@ -0,0 +1,1647 @@
+    }
+#endif // __EDG__
+#endif // defined(MSVC_INTERNAL_TESTING) || defined(__EDG__) || _ITERATOR_DEBUG_LEVEL != 0
+    return true;
+}
+
+template <class CharType = char>
+_CONSTEXPR20_CONTAINER bool test_growth() {
+#if defined(MSVC_INTERNAL_TESTING) || defined(__EDG__) || _ITERATOR_DEBUG_LEVEL != 0 // TRANSITION, VSO-1269894
+    using str = basic_string<CharType>;
+#ifndef __EDG__ // TRANSITION, VSO-1273296
+    {
+        str v(1007, 'a');
+
+        assert(v.size() == 1007);
+        assert(v.capacity() == 1007);
+
+        v.resize(1008);
+
+        assert(v.size() == 1008);
+        assert(v.capacity() == 1510);
+    }
+
+    {
+        str v(1007, 'a');
+
+        assert(v.size() == 1007);
+        assert(v.capacity() == 1007);
+
+        v.resize(8007);
+
+        assert(v.size() == 8007);
+        if constexpr (is_same_v<CharType, char16_t> || is_same_v<CharType, char32_t> || is_same_v<CharType, wchar_t>) {
+            assert(v.capacity() == 8007);
+        } else {
+            assert(v.capacity() == 8015);
+        }
+    }
+
+    {
+        str v(1007, 'a');
+
+        assert(v.size() == 1007);
+        assert(v.capacity() == 1007);
+
+        v.push_back(47);
+
+        assert(v.size() == 1008);
+        assert(v.capacity() == 1510);
+    }
+#if defined(MSVC_INTERNAL_TESTING) || defined(__EDG__) // TRANSITION, VSO-1275530
+    {
+        str v(1007, 'a');
+
+        assert(v.size() == 1007);
+        assert(v.capacity() == 1007);
+
+        str l(3, 47);
+
+        v.insert(v.end(), l.begin(), l.end());
+
+        assert(v.size() == 1010);
+        assert(v.capacity() == 1510);
+    }
+
+    {
+        str v(1007, 'a');
+
+        assert(v.size() == 1007);
+        assert(v.capacity() == 1007);
+
+        str l(7000, 'b');
+
+        v.insert(v.end(), l.begin(), l.end());
+
+        assert(v.size() == 8007);
+        if constexpr (is_same_v<CharType, char16_t> || is_same_v<CharType, char32_t> || is_same_v<CharType, wchar_t>) {
+            assert(v.capacity() == 8007);
+        } else {
+            assert(v.capacity() == 8015);
+        }
+    }
+
+    {
+        str v(1007, 'a');
+
+        assert(v.size() == 1007);
+        assert(v.capacity() == 1007);
+
+        v.insert(v.end(), 3, 'b');
+
+        assert(v.size() == 1010);
+        assert(v.capacity() == 1510);
+    }
+
+    {
+        str v(1007, 'a');
+
+        assert(v.size() == 1007);
+        assert(v.capacity() == 1007);
+
+        v.insert(v.end(), 7000, 'b');
+
+        assert(v.size() == 8007);
+        if constexpr (is_same_v<CharType, char16_t> || is_same_v<CharType, char32_t> || is_same_v<CharType, wchar_t>) {
+            assert(v.capacity() == 8007);
+        } else {
+            assert(v.capacity() == 8015);
+        }
+    }
+#endif // defined(MSVC_INTERNAL_TESTING) || defined(__EDG__)
+#endif // __EDG__
+#endif // defined(MSVC_INTERNAL_TESTING) || defined(__EDG__) || _ITERATOR_DEBUG_LEVEL != 0
+    return true;
+}
+
+int main() {

```

Comment:
I think we've also missed test coverage of the `operator""s` functions -- I see `""sv` literals being used, but no `""s`.